codespell_lib/_codespell.py

```diff
@@ -931,10 +931,15 @@ def main(*args):
                 dirs[:] = [dir_ for dir_ in dirs if not glob_match.match(dir_)]
 
         elif not glob_match.match(filename):  # skip files
-            bad_count += parse_file(
-                filename, colors, summary, misspellings, exclude_lines,
-                file_opener, word_regex, ignore_word_regex, uri_regex,
-                uri_ignore_words, context, options)
+            if not os.path.exists(filename):
+                bad_count += 1
+                print("ERROR: cannot access file: %s" % filename,
```

Comment:
"cannot access" sounds more like a permissions problem, probably better to say "file does not exsit"
```suggestion
                print("ERROR: file does not exist: %s" % filename,
```